waitress/app/models.py

```diff
@@ -21,16 +21,23 @@ def create_user(self, *args, **kwargs):
         """
         return SlackUser(**kwargs)
 
+# Enum fields for user type.
+EMPLOYEE = 'employee'
+GUEST = 'guest'
+
 
 class SlackUser(AbstractBaseUser):
     """
     A class that represents a `SlackUser` account
     """
     id = models.AutoField(unique=True, primary_key=True)
-    slack_id = models.CharField(unique=True, max_length=20)
+    slack_id = models.CharField(unique=True, max_length=20, blank=True)
     firstname = models.CharField(max_length=20,)
     lastname = models.CharField(max_length=20,)
-    email = models.CharField(max_length=60,)
+    email = models.CharField(max_length=60, blank=True)
+    user_type = models.CharField(
```

Comment:
Slack doesn't have us know if a user is a fellow, trainer or staff.
We can only allow for two type at this time: `employee` and `guest`